cmd/repo-updater/repos/store_test.go

```diff
@@ -10,9 +11,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/google/go-cmp/cmp/cmpopts"
-
 	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/inconshreveable/log15"
 	"github.com/opentracing/opentracing-go"
```

Comment:
Since there is a diff, let's fix the grouping in one go
```suggestion
	"github.com/opentracing/opentracing-go"

```